pallets/staking/src/lib.rs

```diff
@@ -2610,6 +2644,36 @@ decl_module! {
             );
             Ok(adjustments)
         }
+
+        /// Start validator slashing from the next era. Only be called by the root.
+        ///
+        /// # Arguments
+        /// * origin - AccountId of root.
+        #[weight = 5_000_000 + T::DbWeight::get().reads_writes(2, 1)]
+        pub fn switch_on_validator_slashing(origin) -> DispatchResult {
+            // Ensure origin should be root.
+            ensure_root(origin)?;
+            // Change the storage by assigning validator slashing status to `true`.
+            SlashingStatus::put(SlashingSwitch::Validator);
+            // Emit event.
+            Self::deposit_event(RawEvent::SlashingOfValidatorOn());
+            Ok(())
+        }
+
+        /// Start validator & nominator slashing from the next era. Only be called by the root.
+        ///
+        /// # Arguments
+        /// * origin - AccountId of root.
+        #[weight = 5_000_000 + T::DbWeight::get().reads_writes(2, 1)]
+        pub fn switch_on_validator_and_nominator_slashing(origin) -> DispatchResult {
+            // Ensure origin should be root.
+            ensure_root(origin)?;
+            // Change the storage by assigning nominator slashing status to `true`.
+            SlashingStatus::put(SlashingSwitch::ValidatorAndNominator);
+            // Emit event.
+            Self::deposit_event(RawEvent::SlashingOfValidatorAndNominatorOn());
+            Ok(())
+        }
     }
```

Comment:
There should be an option to turn off slashing as well. Perhaps we can have a single function that takes the `SlashingSwitch` enum as input.

Reply:
I thought we don't want to switch off slashing once we activate it.

Reply:
We likely won't but I see no harm in keeping out options open.

Reply:
We likely won't but I see no harm in keeping our options open.